### PR TITLE
import_data iiif

### PR DESCRIPTION
**Pull request by eliwaksbaum**
automatically sets the manifest_url field of the manuscript objects. with this command, do we still need a page to display the links a la #478?

before we merge, @napulen :
- What does `@transaction.atomic` do? should it tag the new function as well?
- Where should `manifests.csv` go?

### CODE REVIEW

**Comment by eliwaksbaum** [comment]
would also require an update to the README i guess

**Comment by napulen** [comment]
> would also require an update to the README i guess

yes

**Comment by napulen** [comment]
> * Where should `manifests.csv` go?

In the past, everything has been under `data_dumps`. In lack of a better suggestion (because this is a really unique piece of information), I'd suggest `data_dumps`.

File: .env [hunk 1]
```diff
@@ -7,4 +7,4 @@ PORT=8000
 #Postgres authentication variables
 POSTGRES_DB=cantus_db
 POSTGRES_USER=cantus_admin
-#POSTGRES_PASSWORD=**PROVIDE PASSWORD HERE**
+POSTGRES_PASSWORD=monkeybutt
```
Comment (dchiller) [inline .env:10]: an accidental part of the commit?? 😜😜

File: app/public/cantusdata/management/commands/import_data.py [hunk 2]
```diff
@@ -125,3 +130,16 @@ def import_chant_data(self, **options):
...
+
+    @transaction.atomic
+    def import_iiif_data(self):
+        with open(path.join(BASE_DIR, "data_dumps", "manifests.csv"), "r") as file:
+            csv = file.readlines()
+
+        for row in csv:
+            siglum_manifest = row.split(",")
+            qs = Manuscript.objects.filter(siglum=siglum_manifest[0].strip())
+            if len(qs) > 0:
```
Comment (dchiller) [inline app/public/cantusdata/management/commands/import_data.py:142]: So this is catching the times that the query in line 141 doesn't yield anything (ie when `len(qs) = 0`)?

File: .env (hunk 1 above)
Comment (napulen) [reply .env:10]: I can edit this one out

Comment (eliwaksbaum) [reply .env]: lol i knew i'd forget eventually. good catch

File: app/public/cantusdata/management/commands/import_data.py (hunk 2 above)
Comment (eliwaksbaum) [reply app/public/cantusdata/management/commands/import_data.py:142]: yeah exactly

**Review by napulen — CHANGES_REQUESTED** [review]

**Comment by napulen** [comment]
Cool!

**Pull request merged**